<ui version="4.0" >
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>364</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lImage1" >
       <property name="text" >
        <string>LeftImg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lTitle" >
       <property name="text" >
        <string>&lt;h1>KsirK&lt;/h1></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lImage2" >
       <property name="text" >
        <string>RightImg</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="KPushButton" name="pbNewGame" >
         <property name="text" >
          <string>New Local Game</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="pbJabberGame" >
         <property name="text" >
          <string>Play KsirK over Jabber Network</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="pbNewSocketGame" >
         <property name="text" >
          <string>New Standard TCP/IP Network Game</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="pbJoin" >
         <property name="text" >
          <string>Join Standard TCP/IP Network Game</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KPushButton" name="pbLoad" >
         <property name="text" >
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="KPushButton" name="pbQuit" >
         <property name="text" >
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
